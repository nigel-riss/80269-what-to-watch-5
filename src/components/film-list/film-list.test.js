import React from 'react';
import renderer from 'react-test-renderer';
import FilmList from './film-list.jsx';
import filmsMock from '../../mocks/films.js';


it(`Should FilmList render correctly`, () => {
  const tree = renderer
    .create(
        <FilmList
          films={filmsMock}
        />, {
          createNodeMock: () => {
            return {};
          }
        }
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
