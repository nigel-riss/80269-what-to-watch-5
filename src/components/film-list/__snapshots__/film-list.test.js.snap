// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmList render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "#ffffff",
      }
    }
  >
    <div
      className="small-movie-card__image"
    >
      <video
        disabled={true}
        height={175}
        muted={true}
        poster="/url/to/image.png"
        width={280}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        The Aviator
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "#000000",
      }
    }
  >
    <div
      className="small-movie-card__image"
    >
      <video
        disabled={true}
        height={175}
        muted={true}
        poster="/url/to/image.png"
        width={280}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        The Grand Budapest Hotel
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "#ff00ff",
      }
    }
  >
    <div
      className="small-movie-card__image"
    >
      <video
        disabled={true}
        height={175}
        muted={true}
        poster="/url/to/image.png"
        width={280}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "#ff0000",
      }
    }
  >
    <div
      className="small-movie-card__image"
    >
      <video
        disabled={true}
        height={175}
        muted={true}
        poster="/url/to/image.png"
        width={280}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "#00ff00",
      }
    }
  >
    <div
      className="small-movie-card__image"
    >
      <video
        disabled={true}
        height={175}
        muted={true}
        poster="/url/to/another-image.png"
        width={280}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        What We Do in the Shadows
      </a>
    </h3>
  </article>
</div>
`;
